clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -704,14 +712,22 @@ public CompletedFetch(TopicPartition partition,
      */
     private static class FetchResponseMetricAggregator {
         private final FetchManagerMetrics sensors;
+        private final Map<String, FetchManagerMetrics> topicSensors;
```

Comment:
Is this necessary? There should only ever be one instance of `FetchManagerMetrics`, and it already exposes a method `recordTopicFetchMetrics`.